api/openapi-spec/openapi-spec/README.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes's OpenAPI Specification
```

Comment:
This doesn't seem like a logical file for this PR to be adding, is it a merge problem?

Reply:
likewise for api/openapi-spec/openapi-spec/swagger.json